manual/manual.adoc

```diff
@@ -570,6 +570,12 @@ suggestions:
   easier for you to target. It also encourages monster in-fighting; if done
   effectively, they’ll spend their energy killing each other and you’ll save
   on ammunition.
+  
+* If you encounter a horde of flesh worms or stealth worms, The chainsaw is a
+  great weapon to use. Worms can't attack while being sawed, and if you back 
+  into a corner that is "107 degrees or narrower", they can only come at you 
```

Comment:
This seems very specific. You could probably get away with saying a right angle since 90° is kinda close, and allows for some margin of error.

Reply:
Also, it might me worth mentioning that the chainsaw conserves ammo. Incidentally Freedoom's version of the chainsaw is the "angle grinder".

Reply:
Changed "chainsaw" -> "angle grinder" and reworded to more natural langage in 798d742

Reply:
Reworded again in 1b3188b

Reply:
fixed typo in be6ca27